.travis/s2n_after_travis_build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#  http://aws.amazon.com/apache2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+#
+
+set -ex
+
+# Upload Code Coverage Information to CodeCov.io
+if [[ -n "$CODECOV_IO_UPLOAD" ]]; then
+    bash <(curl -s https://codecov.io/bash) -F ${TESTS};
+fi
```

Comment:
NP: new line missing

Reply:
s2n_travis_build.sh didn't have a trailing newline, so I intentionally matched it.